github-runner-manager/src/github_runner_manager/metrics/runner.py

```diff
@@ -50,21 +50,25 @@
     name="runner_idle_duration_seconds",
     documentation="Time in seconds to runner waiting idle for the job to be picked up.",
     labelnames=[labels.FLAVOR],
+    buckets=[5, 10, 15, 30, 60, 60 * 2, 60 * 3, 60 * 5, 60 * 10, float("inf")],
 )
 RUNNER_QUEUE_DURATION_SECONDS = Histogram(
     name="runner_queue_duration_seconds",
     documentation="Time taken in seconds for the job to be started.",
     labelnames=[labels.FLAVOR],
+    buckets=[5, 30, 60, 60 * 5, 60 * 10, 60 * 20, 60 * 30, 60 * 60, 60 * 60 * 2, float("inf")],
```

Comment:
could we add constants for the minutes and hours? Like `TWO_HOURS_IN_SECONDS = 60 * 60 * 2` ?